pygin/pygin/src/plugin_module.cpp

```diff
@@ -54,6 +55,21 @@ auto EmptyToNull(const wchar_t* Str)
 	return Str && *Str? Str : nullptr;
 }
 
+class plugin
```

Comment:
If this class represents a panel, why not call it panel?

Reply:
Because `OpenW` returns "plugin handle" according to the documentation: https://api.farmanager.com/ru/exported_functions/openw.html
So I assumed it might be used not for Panel API only...

Reply:
Renamed to `context` in 5ef1f612ff8787268b8d48658e3d1e8cad9b7272.